frontend/src/component/project/ProjectAccess/ProjectAccessAssign/ProjectAccessAssign.tsx

```diff
@@ -250,7 +278,11 @@ export const ProjectAccessAssign = ({
                             <span>
                                 {optionUser?.name || optionUser?.username}
                             </span>
-                            <span>{optionUser?.email}</span>
+                            <span>
+                                {optionUser?.name && optionUser?.username
+                                    ? optionUser?.username
+                                    : optionUser?.email}
```

Comment:
Looks like the same logic as above... I'm just wondering if this logic can be encapsulated in a method maybe inside `IUser` as `identifier()` or `accountIdentifier()` which seems to be the purpose of showing either email (for users) or username (for service accounts)

Reply:
Could maybe be represented as a method inside the `User` class or another auxiliary method. Something that would resolve the `IUser` model to a `{ title: string, subtitle: string }` object?

Reply:
I thought about the IUser because that's the type of optionUser, but also because it could be a method of `IAccount` in the future. I feel that `username` or `email` is a way of identifying the user in a more user-friendly way than just `id`, that's why I thought about `accountIdentifier` but I don't know if transforming an IUser to {title, subtiltle} would be a good improvement compared to the current solution